tools/bump-version/basic.js

```diff
@@ -0,0 +1,47 @@
+// import { Octokit } from "@octokit/rest";
+import commandLineArgs from "command-line-args";
+
+import fs from 'fs/promises';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+// https://github.com/75lb/command-line-args
+const optionDefinitions = [
+  { name: 'version', alias: 'v', type: String, required: true },
+];
+
+const cliArgs = commandLineArgs(optionDefinitions);
+
+const appPaths = {};
+appPaths.root = path.join(path.dirname(fileURLToPath(import.meta.url)), "../..");
```

Comment:
Why do it like this instead of constructing the object in one go `{"root": ..., "electronPackage": ...}`